charms/katib-controller/requirements.in

```diff
@@ -1,5 +1,6 @@
-jinja2
+# Version >=0.2.0 contains the base charm code that's needed.
+charmed-kubeflow-chisme >= 0.2.0
+lightkube
 ops
-oci-image
 # Pinning until this resolves https://github.com/yaml/pyyaml/issues/724
```

Comment:
You don't have to change it here, just [FYI](https://github.com/canonical/bundle-kubeflow/issues/843)